src/plugins/inputs/whois.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+exports.commands = ["whois"];
+
+exports.input = function(network, chan, cmd, args) {
+	// This queries server of the other user and not of the current user, which
+	// does not know idle time.
+	// See http://superuser.com/a/272069/208074.
+	network.irc.raw("WHOIS", args[0], args[0]);
```

Comment:
I guess that would be `network.irc.raw("WHOIS", ...args);` in JS with ES6's spread operator. Will wait for a LGTM or better suggestion from @xPaw.